functions/ts/kubeval/third_party/forked/github.com/instrumenta/openapi2jsonschema/.dockerignore

```diff
@@ -0,0 +1,8 @@
+.git/
```

Comment:
No need for "forked" directory.

Also, I assume this PR doesn't contain any changes, just pristine copy.

Reply:
I have dropped "forked" directory.
This PR is just pristine copy, no changes.